README.md

```diff
@@ -1,38 +1,47 @@
 # nvim-comment
 
-Toggle comments in Neovim, using built in **commentstring** filetype option; written in Lua.
-Without a doubt this plugin **is not required** and is a rip off of [TPope's Commentary](https://github.com/tpope/vim-commentary) with less features! What makes this plugin stand out over the numerous other comment plugins written in Lua are:
-
-* Doesn't require nightly build (works on NeoVim 0.4.x)
-* Comments each line, rather than adds block comments; making it easier to toggle code when debugging
-* Uses the built in **commentstring** buffer option to define comment markers
-* Where a marker doesn't have a **space** character as padding this is added, configurable
-** This can be disabled in the options, see below but if useful when workig with numerous linters
-* Supports motions
-
-When the plugin is called it works out the range to comment/uncomment; if all lines in the given range are commented then it uncomments, otherwise it comments the range. This is useful when commenting a block out for testing with a real like comment in it; as for the plugin a comment is a comment.
-
-# Usage
+Toggle comments in Neovim, using built in **commentstring** filetype option;
+written in Lua. Without a doubt this plugin **is not required** and is a rip off
+of [TPope's Commentary](https://github.com/tpope/vim-commentary) with less
+features! What makes this plugin stand out over the numerous other comment
+plugins written in Lua are:
+
+- Comments each line, rather than adds block comments; making it easier to
+  toggle code when debugging
+- Uses the built in **commentstring** buffer option to define comment markers
+- Where a marker doesn't have a **space** character as padding this is added,
+  configurable \*\* This can be disabled in the options, see below but if useful
+  when working with numerous linters
+- Supports motions
+- Dot `.` repeatable
+
+When the plugin is called it works out the range to comment/uncomment; if all
+lines in the given range are commented then it uncomments, otherwise it comments
+the range. This is useful when commenting a block out for testing with a real
+like comment in it; as for the plugin a comment is a comment.
+
+## Usage
 
 Either use the command `CommentToggle`, e.g.:
 
-* `CommentToggle` comment/uncomment current line
-* `67,69CommentToggle` comment/uncomment a range
-* `'<,'>CommentToggle` comment/uncomment a visual selection
+- `CommentToggle` comment/uncomment current line
```

Comment:
Why change these?

Reply:
All README changes were dictated by linter. I think in above snippet `*` is redundant. It's either `*` or `-` list.

Reply:
ah I don't use a markdown linter, shame on me.